nextflow.config

```diff
@@ -85,15 +85,15 @@ params {
     expected_taxon       = null     // optional: provide expected organism
 
     // Defualt dbs
-    midas_db             = "s3://tgn-latch-dev-database-2025/midas/midas_db_v1.2.tar.gz"
-    kraken_db            = "s3://tgn-latch-dev-database-2025/kraken2/terra_databases_kraken2_kraken2.kalamari_5.1.tar"
-    kmerfinder_db        = "s3://tgn-latch-dev-database-2025/kmerfinder/kmerfinder_bacteria_20230911.tar.gz"
-    gambit_db_genomes    = "s3://tgn-latch-dev-database-2025/gambit/gambit-metadata-2.0.1-20250505.gdb" 
-    gambit_db_signatures = "s3://tgn-latch-dev-database-2025/gambit/gambit-signatures-2.0.1-20250505.gs"
-    bakta_db_path        = "s3://tgn-latch-dev-database-2025/bakta/"
+    midas_db             = "latch://tgn-latch-dev-database-2025.mount/midas/midas_db_v1.2.tar.gz"
```

Comment:
with this config, non latch users will run into errors, probably they should be defined in parameters.py instead